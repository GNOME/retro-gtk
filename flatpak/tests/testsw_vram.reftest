[Retro Reftest]
Path=/testsw_vram
Core=/app/lib/libretro/testsw_vram_libretro.so

[Frame 0]
State=Refresh
Video=testsw_vram.png
