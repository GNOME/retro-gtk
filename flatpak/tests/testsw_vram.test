[Retro Reftest]
Path=/testsw_vram
Core=/app/lib/libretro/testsw_vram_libretro.so

[Frame 0]
Tests=Run;Video;StateRefresh;
Video=testsw_vram.png
