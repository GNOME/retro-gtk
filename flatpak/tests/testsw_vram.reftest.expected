/testsw_vram/Boot: OK
/testsw_vram/Options: OK
/testsw_vram/0/State Refresh: OK
/testsw_vram/0/Run: OK
/testsw_vram/0/Video: OK
